### ISSUE TITLE
Ideas

### ISSUE THREAD

**Issue by ankane** [ankane/quirk#1]
- [x] When RMSLE is eval metric, have XGBoost optimize for it
- [ ] Improve technical performance
- [ ] Improve model performance
- [ ] Hyper-parameter tuning
- [ ] Text features
- [ ] Geo features
- [ ] Name and address features
